SPECS/prometheus-node-exporter/prometheus-node-exporter.spec

```
@@ -58,7 +58,11 @@ LDFLAGS="-X github.com/prometheus/common/version.Version=%{version}      \
          -X github.com/prometheus/common/version.Branch=tarball          \
          -X github.com/prometheus/common/version.BuildDate=%{build_date} \
          -X github.com/ncabatoff/process-exporter/version.GoVersion=%{go_version}"
-go build -ldflags "$LDFLAGS" -mod=vendor -v -a -tags "$BUILDTAGS" -o bin/node_exporter ./collector
+go build -ldflags "$LDFLAGS" -mod=vendor -v -a -tags "$BUILDTAGS" -o bin/node_exporter
+
+%check
+make test
+bin/node_exporter --help
```

In my opinion running this command would override the results from the `make test`.
Even if the tests fail, our tooling wont be able to catch it, as it depends on the exit status of the last thing that runs in the check section.
IMO we should swap them:
```suggestion
bin/node_exporter --help
make test
```

---

Thank you for the suggestion!

It's kind of sad that only the last thing has its exit status checked. Would something like this be more appropriate to check both things?

```suggestion
bin/node_exporter --help && make test
```

---

Yep, this looks good to me :)

---

Done :)